rust/capture/src/v0_endpoint.rs

```diff
@@ -311,13 +313,16 @@ pub fn process_single_event(
     // if this event was historical but not assigned to the right topic
     // by the submitting user (i.e. no historical prop flag in event)
     // we should route it there using event#now if older than 1 day
-    let dayold_event = {
+    let dayold_event = if historical_cfg.enable_historical_rerouting {
         if let Ok(event_dt) = DateTime::parse_from_rfc3339(&event.now) {
```

Comment:
question: This looks like the wall clock, not the event timestamp. If yes, it won't get picked up by the threshold.

https://github.com/PostHog/posthog/blob/588c07a1e407bb0447ce23ec82ec166ee16b8fa6/rust/capture/src/v0_endpoint.rs#L116